runtime/development/src/lib.rs

```diff
@@ -2219,7 +2225,7 @@ construct_runtime!(
 		BaseFee: pallet_base_fee::{Pallet, Call, Config<T>, Storage, Event} = 162,
 		Ethereum: pallet_ethereum::{Pallet, Config<T>, Call, Storage, Event, Origin} = 163,
 		EthereumTransaction: pallet_ethereum_transaction::{Pallet, Storage} = 164,
-		LiquidityPoolsAxelarGateway: axelar_gateway_precompile::{Pallet, Call, Storage, Event<T>} = 165,
+		AxelarRouter: pallet_axelar_router::{Pallet, Call, Storage, Event<T>} = 165,
```

Comment:
Reusing index, so we should remove the previous Axelar state on-chain.

Also, another TODO: we should remove the code from axelar and routers crates

Reply:
I have added it to the post-audit TODO list.